packages/terra-list/src/ListSectionHeader.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Arrange from 'terra-arrange';
+import styles from './ListSectionHeader.scss';
+
+const cx = classNames.bind(styles);
+
+const KEYCODES = {
+  ENTER: 13,
+  SPACE: 32,
+};
+
+const propTypes = {
+  /**
+   * Text to be displayed on the ListSectionHeader.
+   */
+  title: PropTypes.string,
+  /**
+   * Callback function triggered when the accordion icon is clicked.
+   */
+  onClick: PropTypes.func,
+  /**
+   * Whether the accordion icon should be displayed in its 'open' or 'closed' position.
+   */
+  isOpen: PropTypes.bool,
+  /**
+   * Optionally sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`. Default `level=2`.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
+};
+
+const defaultProps = {
+  title: '',
```

Comment:
I believe we made the decision to no longer default string props to be empty strings. They should be `undefined` instead. @StephenEsser correct me if I'm remembering this wrong.

Reply:
This is correct.